src/js/components/Widgets/SignInModal.jsx

```diff
@@ -181,15 +215,7 @@ be honored, Cordova tries to do the best it can, but sometimes it crashes and lo
 For Cordova eliminate as many fixed vertical dimensions as needed to avoid overconstraint.
 */
 const styles = theme => ({
-  dialogRoot: isWebApp() ? {
-    height: '100%',
-    // position: 'absolute !important', // Causes problem on Firefox
-    top: '-15%',
-    left: '0% !important',
-    right: 'unset !important',
-    bottom: 'unset !important',
-    width: '100%',
-  } : {
+  dialogRoot: {
```

Comment:
I think we still need this "isWebApp" conditional to turn off the `position: 'absolute !important'` to get rid of an error I found in Firefox/WebApp Browser.

Reply:
I'll back out this change, ill need to make a full testing pass anyways for cordova.